auth-service/src/test/kotlin/uk/gov/communities/delta/security/OAuthSSOLoginTest.kt

```diff
@@ -324,14 +333,17 @@ class OAuthSSOLoginTest {
             ldapUserLookupServiceMock = mockk<UserLookupService>()
             registrationService = mockk<RegistrationService>()
             organisationService = mockk<OrganisationService>()
+            userAuditService = mockk<UserAuditService>()
+            loginCounter = mockk<Counter>()
```

Comment:
Could this be called SSO login counter just to make it obvious that it is different from the "successfulLoginNoopCounter" counter used for the loginPageController?